web/javascript/torrent-row.js

```diff
@@ -107,6 +107,14 @@ TorrentRendererHelper.formatDL = function (t) {
     return '↓ ' + Transmission.fmt.speedBps(t.getDownloadSpeed());
 };
 
+TorrentRendererHelper.formatETA = function(t)
+{
+        var eta = t.getETA();
+        if (eta < 0 || eta >= (999 * 60 *60))
+                return "na";
```

Comment:
What I'm saying is that "na" or "N/A" alone doesn't make sense to me, as it's not clear what exactly is not available. If it will be "ETA: N/A" then it will be clear as day ;)